platform/getting-started-with-particular-service-platform.md

```diff
@@ -8,6 +8,8 @@ tags:
 - Visual Studio
 ---
 
+include: sm-discontinued
```

Comment:
I think we should move this file under the am directory. And also rename it to "getting started with am". And also move it in the menu